source/index.html.erb

```diff
@@ -46,6 +46,26 @@ per_page: 4
   </div>
 </div>
 
+<div class="featured-tip">
+  <div class="container">
+    <% blog.articles.select{ |article| article.data.category == 'tips' }[0...1].each do |article| %>
+      <div class="left">
+        <h4><%= article.data.pre_title %></h4>
+        <h2><%= article.data.title %></h2>
```

Comment:
You might not want to skip `h` levels. Here's some food for thought...

https://www.hobo-web.co.uk/headers/

Reply:
The h4 should actually be an h5, but I tried to keep the hierarchy of the h's relative to size

![screen shot 2017-10-18 at 5 41 37 pm](https://user-images.githubusercontent.com/13579950/31744137-a09ff970-b42b-11e7-8dd4-65f87548273d.png)